ivy/container/data_type.py

```diff
@@ -567,6 +567,35 @@ def is_float_dtype(
         prune_unapplied: bool = False,
         map_sequences: bool = False,
     ) -> ivy.Container:
+        """
+        `ivy.Container` instance method variant of `ivy.is_float_dtype`.
+        This method simply wraps the function,
+        and so the docstring for `ivy.is_float_dtype`
+        also applies to this method with minimal changes.
+
+        Parameters
+        ----------
+        key_chains : Union[List[str], Dict[str, str]]
```

Comment:
The self parameter is needed here.
You should also add docstrings to the static function (I think it is above this instance method).

Reply:
You should also add docstrings to the static function (I think it is above this instance method).

Reply:
Both of these should have been resolved in the latest commit, no?